.eslintrc.js

```diff
@@ -28,6 +28,28 @@ module.exports = {
     // allow async-await
     'generator-star-spacing': 'off',
     // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    // TODO enable the rules to achieve lint standard consistency towards projects
```

Comment:
Hi @rubyllamadora I have created a ticket for this TODO https://digital-engagement.atlassian.net/browse/SDPA-4970